examples/creating_a_gwcs_from_quantities.py

```diff
@@ -0,0 +1,50 @@
+"""
+===============================================
+How to create an gWCS from quantities and times
+===============================================
+
+This example shows to create a gWCS from quantities in this example time and energy.
+
+"""
+
+import numpy as np
+
+import astropy.units as u
+from astropy.time import Time
+
+from ndcube import ExtraCoords, NDCube
+from ndcube.extra_coords import QuantityTableCoordinate, TimeTableCoordinate
+
+##############################################################################
+# First we create our coordinate arrays.
+
+energy = np.arange(10) * u.keV
+time = Time('2020-01-01 00:00:00') + np.arange(9)*u.s
+
+##############################################################################
+# Then we use use
+# `~ndcube.extra_coords.table_coord.QuantityTableCoordinate` and
+# `~ndcube.extra_coords.table_coord.TimeTableCoordinate` to create table coordinates.
+
+energy_coord = QuantityTableCoordinate(energy, names='energy', physical_types='em.energy')
+energy_coord
```

Comment:
I think these lines need to be printed otherwise in the terminal, there will be no output.